packages/gatsby-remark-images/src/constants.js

```diff
@@ -1,3 +1,13 @@
+exports.defaults = {
```

Comment:
`exports.defaults` might be confused with `export default` so maybe rename it to something verbose like `DEFAULT_OPTIONS`.
```suggestion
exports.DEFAULT_OPTIONS= {
```